core/contracts/financial_templates/implementation/Liquidatable.sol

```diff
@@ -45,9 +45,19 @@ contract Liquidatable is PricelessPositionManager {
         FixedPoint.Unsigned settlementPrice; // Final price as determined by an Oracle following a dispute
     }
 
+    // Mutable data structures.
+
     // Liquidations are unique by ID per sponsor
     mapping(address => LiquidationData[]) public liquidations;
 
+    // Total collateral in liquidation.
+    FixedPoint.Unsigned public rawLiquidationCollateral;
+
+    // Adjustment parameter applied to locked collateral to compensate for fees.
+    FixedPoint.Unsigned public liquidationFeeAdjustment;
```

Comment:
How is this used?

Reply:
Oh thanks for pointing that out. It's not -- I forgot to remove it.

Reply:
Done.